contracts/stake-cw20-external-rewards/src/state.rs

```diff
@@ -0,0 +1,32 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::{Addr, Uint128, Uint256};
+use cw20::Denom;
+
+use cw_storage_plus::{Item, Map};
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub struct Config {
+    pub owner: Option<Addr>,
```

Comment:
Is there a reason for both an owner and manager? Are there certain actions an owner can do and a manager cannot?

Reply:
yes, main difference is owner can change both the owner and manager address. Manager can not alter the owner. The idea here is a DAO might want to delegate the management of the contract to an operations multisig. The owner/manager model allows this delegation will still leaving the dao the ability to reassert control if the manager ever misbehaves.

Reply:
Awesome, I noticed these are documented under the InstantiateMsg, so I'm happy with how it is.